plexapi/library.py

```diff
@@ -2033,6 +2040,26 @@ def sync(self, bitrate, limit=None, **kwargs):
         kwargs['policy'] = Policy.create(limit)
         return super(MusicSection, self).sync(**kwargs)
 
+    def sonicAdventure(
+            self,
+            startID: int,
+            endID: int,
+            **kwargs: Any,
+    ) -> list[Track]:
```

Comment:
I think `sonicAdventure()` should accept both `Track` objects and rating keys as arguments.

`sonicAdventure(start=track1, end=track2)`

Are `kwargs` necessary? What `kwargs` are expected?

Reply:
> Are kwargs necessary? What kwargs are expected?
any kwargs that could be passed to fetchitems, maybe a case where one needs to use filters to filter out tracks based on some param, like DJ stretch but with some other filtering mechanism.